### PR TITLE
tests: verify /etc release symlinks are valid

### PR DESCRIPTION
**Pull request by HuijingHei**
See:
- https://bugzilla.redhat.com/show_bug.cgi?id=2068148
- https://github.com/openshift/os/pull/815

### CODE REVIEW

**Review by ravanelli — DISMISSED** [review]
lgtm

**Comment by mike-nguyen** [comment]
LGTM

File: tests/kola/files/check-symlink [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# Verify the symlink pointing to a valid file
+# See:
+# - https://bugzilla.redhat.com/show_bug.cgi?id=2068148
+# - https://github.com/openshift/os/pull/815
+
+set -xeuo pipefail
+
+. $KOLA_EXT_DATA/commonlib.sh
+
+for file_name in /etc/system-release-cpe /etc/system-release /etc/redhat-release
```
Comment (jlebon) [inline tests/kola/files/check-symlink]: While we're here, we might as well add `/etc/os-release` too.

**Review by jlebon — COMMENTED** [review]
This mostly LGTM, but can I suggest a more descriptive commit title? E.g. maybe something like `tests: verify /etc release symlinks are valid`?

**Review by jlebon — APPROVED** [review]
Thanks!

**Comment by travier** [comment]
I'm late to the party but is there any reason preventing us from testing basically all symlinks in the image?

**Comment by HuijingHei** [comment]
> is there any reason preventing us from testing basically all symlinks in the image?

Good question, do we need to fix all broken symlinks if found?

Just check /etc and find there are 4 symlinks that are broken:
```
[core@cosa-devsh ~]$ sudo find /etc -xtype l | xargs ls -l
lrwxrwxrwx. 1 root root 29 Jun  9 17:03 /etc/alternatives/ncman -> /usr/share/man/man1/ncat.1.gz
lrwxrwxrwx. 1 root root 43 Jun  9 17:03 /etc/alternatives/unversioned-python-man -> /usr/share/man/man1/unversioned-python.1.gz
lrwxrwxrwx. 1 root root 31 Jun  9 17:03 /etc/grub2-efi.cfg -> ../boot/efi/EFI/redhat/grub.cfg
lrwxrwxrwx. 1 root root 15 Jun  9 17:03 /etc/sysconfig/grub -> ../default/grub
[core@cosa-devsh ~]$ ls -l /usr/share/man/man1/ncat.1.gz /usr/share/man/man1/unversioned-python.1.gz ../boot/efi/EFI/redhat/grub.cfg ../default/grub
ls: cannot access '/usr/share/man/man1/ncat.1.gz': No such file or directory
ls: cannot access '/usr/share/man/man1/unversioned-python.1.gz': No such file or directory
ls: cannot access '../boot/efi/EFI/redhat/grub.cfg': No such file or directory
ls: cannot access '../default/grub': No such file or directory
```

**Comment by travier** [comment]
I think we should. If they are broken we should fix them or remove them.

**Comment by travier** [comment]
And I would check both etc & usr for broken symlinks

**Comment by travier** [comment]
Made https://github.com/coreos/fedora-coreos-config/issues/1782 for tracking

**Pull request merged**